### PR TITLE
docs: Add steps to configure the service network #85

### PR DESCRIPTION
**Pull request by runkecheng**
- Replace the picture in the document
- add steps to configure the service network 
   - NodePort
   - LoadBalancer

### CODE REVIEW

**Review by andyli029 — COMMENTED** [review]
Fix the commit log

**Comment by molliezhang** [comment]
LGTM

File: docs/KubeSphere/deploy_radondb-mysql_on_kubesphere.md [hunk 1]
```diff
@@ -153,74 +153,91 @@ helm package charts
...
 
-![应用配置](png/应用配置.png)
+  ![应用配置](png/应用配置.png)
 
   >说明：示例使用默认参数部署。
 
 #### **步骤 3：部署校验**
 
-在**项目管理**管理中心，选择 **应用负载 > 工作负载**，并选择**有状态副本集**页签，可查看到release 名为 `my-release` 的副本集，则 radondb-mysql 集群已成功部署。
+在**项目管理**管理中心，选择 **应用负载 > 工作负载**，并选择**有状态副本集**页签，可查看到 release 名为 `demo` 的副本集，则 radondb-mysql 集群已成功部署。
```
Comment (runkecheng) [inline docs/KubeSphere/deploy_radondb-mysql_on_kubesphere.md:167]: radondb-mysql -> RadonDB MySQL

File: docs/KubeSphere/deploy_radondb-mysql_on_kubesphere.md [hunk 2]
```diff
@@ -148,79 +148,96 @@ helm package charts
 
 ![应用负载](png/应用负载.png)
 
-  点击 **部署新应用**，在 **来自应用模板**窗口中选择 `radondb-mysql`。
+  点击 **部署新应用**，在 **来自应用模板**窗口中选择 `RadonDB MySQL`。
```
Comment (zhyass) [inline docs/KubeSphere/deploy_radondb-mysql_on_kubesphere.md:151]: From the [picture](https://github.com/runkecheng/radondb-mysql-kubernetes/blob/rename_keyword/docs/KubeSphere/png/%E9%83%A8%E7%BD%B2%E6%96%B0%E5%BA%94%E7%94%A8.png), the application name is `radondb-mysql`

File: docs/KubeSphere/deploy_radondb-mysql_on_kubesphere.md [hunk 3]
```diff
@@ -148,79 +148,96 @@ helm package charts
...
 
-![应用配置](png/应用配置.png)
+  ![应用配置](png/应用配置.png)
 
   >说明：示例使用默认参数部署。
 
 #### **步骤 3：部署校验**
 
-在**项目管理**管理中心，选择 **应用负载 > 工作负载**，并选择**有状态副本集**页签，可查看到release 名为 `my-release` 的副本集，则 radondb-mysql 集群已成功部署。
+在**项目管理**管理中心，选择 **应用负载 > 工作负载**，并选择**有状态副本集**页签，可查看到 release 名为 `demo` 的副本集，则 RadonDB MySQL 集群已成功部署。
```
Comment (zhyass) [inline docs/KubeSphere/deploy_radondb-mysql_on_kubesphere.md:167]: The name of the statefulset is `demo-radondb-mysql`
https://github.com/runkecheng/radondb-mysql-kubernetes/blob/rename_keyword/docs/KubeSphere/png/%E6%8E%A7%E5%88%B6%E5%8F%B0%E9%83%A8%E7%BD%B2%E6%88%90%E5%8A%9F.png

File: docs/KubeSphere/deploy_radondb-mysql_on_kubesphere.md [hunk 4]
```diff
@@ -148,79 +148,96 @@ helm package charts
...
-RadonDB MySQL 由一个主节点和两个从节点组成，可通过如下命令访问每个 RadonDB MySQL 节点：
+您需准备一个用于连接 RadonDB MySQL 的客户端。
 
-```txt
-<pod-name>.my-release-radondb-mysql
-```
+### **客户端与 RadonDB MySQL 在同一项目中**
 
-## **连接数据库**
+当客户端和 RadonDB MySQL 集群在同一个项目中时，可使用 leader/follower 代替具体的 ip 和端口。
```
Comment (zhyass) [inline docs/KubeSphere/deploy_radondb-mysql_on_kubesphere.md:177]: Maybe you can display the `服务` before `连接数据库`, and briefly describe the meaning of each service. 
This is not `leader/follower`, but the `leader service` and `follower service`.

File: docs/KubeSphere/deploy_radondb-mysql_on_kubesphere.md [hunk 5]
```diff
@@ -148,79 +148,96 @@ helm package charts
...
 
-### **步骤 1：创建 Client**
+- 连接主节点。
```
Comment (zhyass) [inline docs/KubeSphere/deploy_radondb-mysql_on_kubesphere.md:179]: underline  read write

File: docs/KubeSphere/deploy_radondb-mysql_on_kubesphere.md [hunk 6]
```diff
@@ -148,79 +148,96 @@ helm package charts
...
+   ```bash
+   mysql -h <release名称>-radondb-mysql-leader -u <用户名> -p
+   ```
 
-创建一个用于连接 RadonDB MySQL 集群的客户端主机。
+- 连接从节点。
```
Comment (zhyass) [inline docs/KubeSphere/deploy_radondb-mysql_on_kubesphere.md:184]: underline read only

File: docs/KubeSphere/deploy_radondb-mysql_on_kubesphere.md [hunk 7]
```diff
@@ -148,79 +148,96 @@ helm package charts
...
+
+     在 `service.beta.kubernetes.io/qingcloud-load-balancer-eip-ids` 参数中填写可用的 EIP ID，系统会自动为 EIP 创建负载均衡器和对应的监听器。
+
+     在 `service.beta.kubernetes.io/qingcloud-load-balancer-type` 参数中填写负载均衡器承载能力类型，具体可查看 [CreateLoadBalancer](https://docs.qingcloud.com/product/api/action/lb/create_loadbalancer.html)。
+
+     点击确定自动生成转发端口，在 KubeSphere 集群同一网络内可通过集群IP/节点IP和此端口访问服务。
+
+      ![负载均衡端口](png/负载均衡端口.png)
+
+3. 获取外网访问 IP 和端口。
```
Comment (zhyass) [inline docs/KubeSphere/deploy_radondb-mysql_on_kubesphere.md:229]: Only applicable to scenarios where kubesphere is created in the qingcloud console. 
In fact, we do not recommend port forwarding on the public network.  You can `在同一 VPC 下主机来访问，或者通过青云 VPN 服务来访问`.

File: docs/Kubernetes/deploy_radondb-mysql_on_kubernetes.md [hunk 8]
```diff
@@ -28,18 +28,18 @@ git clone https://github.com/radondb/radondb-mysql-kubernetes.git
 
 > release 是运行在 Kubernetes 集群中的 Chart 的实例。通过命令方式部署，需指定 release 名称。
 
-以下命令指定 release 名为 `my-release`。
+以下命令指定 release 名为 `demo`。
```
Comment (zhyass) [inline docs/Kubernetes/deploy_radondb-mysql_on_kubernetes.md:31]: Refer to the suggestions above

**Review by zhyass — CHANGES_REQUESTED** [review]

File: docs/KubeSphere/deploy_radondb-mysql_on_kubesphere.md [hunk 9]
```diff
@@ -34,9 +34,36 @@ RadonDB MySQL 是基于 MySQL 的开源、高可用、云原生集群解决方
 
 ## **部署步骤**
 
-可选择 [命令行](#通过命令行部署-RadonDB-MySQL-集群) 或 [控制台](#通过控制台部署-RadonDB-MySQL-集群) 两种方式部署 RadonDB MySQL 集群。
+可通过如下三种方式部署 RadonDB MySQL 集群。
 
-### **通过命令行部署 RadonDB MySQL 集群**
+- [通过 Helm repo 部署 RadonDB MySQL 集群](#通过-Helm-repo-部署-RadonDB-MySQL-集群)
```
Comment (hustjieke) [inline docs/KubeSphere/deploy_radondb-mysql_on_kubesphere.md:39]:
```
- [通过 Helm repo 部署 RadonDB MySQL 集群](#通过-Helm-repo-部署-RadonDB-MySQL-集群)
- [通过 Git 部署 RadonDB MySQL 集群](#通过-Git-部署-RadonDB-MySQL-集群)
- [通过控制台部署 RadonDB MySQL 集群](#通过控制台部署-RadonDB-MySQL-集群)
```
split to three files, like 
```
- [通过 Helm repo 部署 RadonDB MySQL 集群](link to filename1)
- [通过 Git 部署 RadonDB MySQL 集群](link to filename2)
- [通过控制台部署 RadonDB MySQL 集群](link to filename3)
```

File: docs/KubeSphere/deploy_radondb-mysql_on_kubesphere.md [hunk 10]
```diff
@@ -34,9 +34,36 @@ RadonDB MySQL 是基于 MySQL 的开源、高可用、云原生集群解决方
...
+- [通过控制台部署 RadonDB MySQL 集群](#通过控制台部署-RadonDB-MySQL-集群)
+
+### **通过 Helm repo 部署 RadonDB MySQL 集群**
+
+#### **步骤 1 : 添加仓库**
+
+添加并更新 helm 仓库。
+
+```bash
+helm repo add <repo 名称> https://charts.kubesphere.io/test
```
Comment (hustjieke) [inline docs/KubeSphere/deploy_radondb-mysql_on_kubesphere.md:50]: do not use `<repo 名称>`,   change it to a real name   `test`

File: docs/KubeSphere/deploy_radondb-mysql_on_kubesphere.md (hunk 9 above)
Comment (runkecheng) [reply docs/KubeSphere/deploy_radondb-mysql_on_kubesphere.md:39]: next pr

**Comment by hustjieke** [comment]
LGTM

**Pull request closed without merging**